coderd/httpmw/apikey.go

```diff
@@ -369,13 +369,15 @@ func ExtractAPIKey(rw http.ResponseWriter, r *http.Request, cfg ExtractAPIKeyCon
 		// If the API Key is associated with a user_link (e.g. Github/OIDC)
 		// then we want to update the relevant oauth fields.
 		if link.UserID != uuid.Nil {
-			// nolint:gocritic
+			// nolint:gocritic // system needs to update user link
```

Comment:
```suggestion
			//nolint:gocritic // system needs to update user link
```

This, I believe is the standard, but we have mixed use 😔 `//nolint` 289 vs `// nolint` 178.